src/Clients/SymfonyClient.php

```diff
@@ -41,7 +43,9 @@ public function request(Service $service, Place $place): string
             $clientParameters = array_merge($service->getParameters(), $options);
             unset($clientParameters['method']);
 
-            return $this->httpClient->request($method, $service->getURI(), $clientParameters)->getContent();
+            $symfonyResponse = $this->httpClient->request($method, $service->getURI(), $clientParameters);
+
+            return ResponseConverter::convertToPsr7($symfonyResponse);
```

Comment:
damn you,  Symfony Core team :'(

Reply:
Maybe the PSR7 didn't exist at the time?

Reply:
Hmm, I don't think so as the HttpClient component has been shipped in Symfony 4.3 :(